manifest.md

```diff
@@ -188,15 +187,15 @@ Here is an example manifest for a typical artifact:
 }
 ```
 
-Here is an example manifest for a simple artifact without content in the config, using the scratch descriptor:
+Here is an example manifest for a simple artifact without content in the config, using the filler descriptor:
 
 ```json,title=Artifact%20without%20config&mediatype=application/vnd.oci.image.manifest.v1%2Bjson
 {
   "schemaVersion": 2,
   "mediaType": "application/vnd.oci.image.manifest.v1+json",
   "artifactType": "application/vnd.example+type",
   "config": {
-    "mediaType": "application/vnd.oci.scratch.v1+json",
+    "mediaType": "application/vnd.oci.filler.v1+json",
     "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
     "size": 2
   },
```

Comment:
It's not _super_ important, but it seems like a good idea for us to be consistent in how we order these (between the struct and all our examples).  Perhaps that's a good follow-up PR. :smile:

Reply:
Might as well fix it now. Done in the latest update.